src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -71,8 +103,14 @@ export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProp
         // todo
     };
 
-    private copyFailureDetails = (): void => {
-        // todo
+    private copyFailureDetails = (event: React.MouseEvent<any>): void => {
+        const onClick = this.props.deps.detailsViewActionMessageCreator.copyIssueDetailsClicked;
+        if (onClick) {
+            onClick(event);
+        }
+        // tslint:disable-next-line: no-floating-promises
+        this.props.deps.navigator.copyToClipboard('The quick brown fox jumps over the lazy dog'); // to be changed when we finish the new data format and builder
```

Comment:
make this a promise call and use await to make sure that the toast appears after the copy action finishes.

Reply:
this can't be promise because onClick should be either boolean or void but I will find another solution for it